src/backend/src/lib.rs

```diff
@@ -580,14 +581,14 @@ fn get_user_profile() -> Result<UserProfile, GetUserProfileError> {
 #[query(guard = "caller_is_allowed")]
 #[allow(clippy::needless_pass_by_value)]
 fn get_users(request: GetUsersRequest) -> GetUsersResponse {
```

Comment:
unrelated to this PR but, it crossed my mind: should the endpoint actually be named `list_users` instead of `get_users`?

Reply:
Done